config/labels/analyzers/clangsa.json

```diff
@@ -402,12 +396,14 @@
       "profile:default",
       "profile:extreme",
       "profile:sensitive",
-      "sei-cert:arr30-c",
-      "sei-cert:dcl38-c",
+      "sei-cert:exp34-c",
       "severity:HIGH"
     ],
     "core.StackAddrEscapeBase": [
       "guideline:sei-cert",
+      "sei-cert:dcl30-c",
+      "sei-cert:exp54-cpp",
+      "sei-cert:exp61-cpp",
```

Comment:
Alphabetic order should be adhered to because tooling that affects the label files will always dump them in a sorted fashion, to reduce the number of conflicts and useless changing lines later on.

Reply:
The cert rule tags are, but the entire list of labels for many checkers aren't.

`doc_url` < `guideline` < `profile` < `sei-cert` < `severity`

Reply:
Can we leave label ordering to a later commit?

Reply:
Yeah, I'll rebase #4224.